src/@types/index.ts

```diff
@@ -0,0 +1,78 @@
+export interface SpriteBox {
```

Comment:
Because the refactor touched so many files, I preferred having some types to help me during development.